codemods/jscodeshift/immutable/4/isterable-to-iscollection/config.json

```diff
@@ -0,0 +1,7 @@
+{
+	"schemaVersion": "1.0.0",
+	"name": "immutable/4/isterable-to-iscollection",
```

Comment:
you need to add this codemod to the codemod set in the `/sets/` directory.

Reply:
resolved in #25